mars/services/task/execution/ray/executor.py

```diff
@@ -118,29 +124,66 @@ def execute_subtask(
     subtask_id: str,
     subtask_chunk_graph: ChunkGraph,
     output_meta_keys: Set[str],
-    input_keys: List[str],
     *inputs,
 ):
-    logger.info("Begin to execute subtask: %s", subtask_id)
+    """The function used for execute subtask in ray task.
+    If subtask is shuffle mapper, no chunk meta will be returned, otherwise return chunk mata."""
     ensure_coverage()
     subtask_chunk_graph = deserialize(*subtask_chunk_graph)
-    # inputs = [i[1] for i in inputs]
+    subtask_digraph = subtask_chunk_graph.to_dot()
```

Comment:
Can you dump the subtask graph dots if error occurred? We want the success execution branch as fast as possible.

Reply:
Fixed